docs/Change_Log.md

```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+## Updated Command Permissions
+
+#### Apr 27, 2022
+
+Application command permissions have been updated to add more granular control and access to commands in Discord. You can read the major changes below, and [the updated documentation](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/permissions) for details.
+
+#### Breaking changes
+
+- Bearer tokens are now required to edit command permissions. Bearer tokens are tokens tied to an authenticating user's permissions, and can be [retrieved using OAuth](#DOCS_TOPICS_OAUTH2). The user must have permission to manage the guild and roles.
+- [`applications.commands.permissions.update`](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-scopes) scope is now required to edit command permissions
+- Deprecates batch editing endpoint ([`PUT /applications/{application.id}/guilds/{guild.id}/commands/permissions`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/batch-edit-application-command-permissions))
```

Comment:
```suggestion
- The batch editing endpoint ([`PUT /applications/{application.id}/guilds/{guild.id}/commands/permissions`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/batch-edit-application-command-permissions)) has been deprecated.
```

I think that it should be updated to match the tense of the remaining breaking changes.

Reply:
also, pretty sure this endpoint has been removed, not deprecated